.github/workflows/auto_build.yml

```
@@ -0,0 +1,233 @@
+        name: wrapper-apk
+        path: "${{ env.WRAPPER_APK_PATH }}/${{ steps.detect-project.outputs.BUILD_TYPE }}/*.apk"
+        retention-days: 7
+
+  final-result:
+    name: Build Summary
+    needs: detect-and-build
+    runs-on: ubuntu-latest
+    if: always()
+    
+    steps:
+    - name: Check build status
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)